ducktape/tests/runner.py

```diff
@@ -73,10 +73,23 @@ def run_all_tests(self):
 
             # Create single testable unit and corresponding test result object
             self.current_test_context = test_context
+            result = TestResult(self.current_test_context)
+
+            if self.current_test_context.ignore:
+                # Skip running this test, but keep track of the fact that we ignored it
```

Comment:
Interesting, I was taking `ignore` to mean that you should run the test, but because we know it might fail you don't want it to mark the entire test run as having failed. This approach could work too, but it also means that any transient test failures that may be hard to reproduce can't get additional logging added, which could be valuable since it allows you to keep collecting info from normal test executions without having to pull up your own test infrastructure.

Not sure which is more useful.